patch-deps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I was trying out a fork of embroider to try to get transpilation without ember-cli-typescript -- some of that effort here: https://github.com/embroider-build/embroider/pull/895